com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -38,231 +30,317 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem
 
 #pragma warning restore IDE1006 // restore naming rule violation check
 
-#if DEVELOPMENT_BUILD || UNITY_EDITOR
-        private static ProfilerMarker s_SyncTime = new ProfilerMarker($"{nameof(NetworkManager)}.SyncTime");
-        private static ProfilerMarker s_TransportPoll = new ProfilerMarker($"{nameof(NetworkManager)}.TransportPoll");
-        private static ProfilerMarker s_TransportConnect = new ProfilerMarker($"{nameof(NetworkManager)}.TransportConnect");
-        private static ProfilerMarker s_HandleIncomingData = new ProfilerMarker($"{nameof(NetworkManager)}.{nameof(HandleIncomingData)}");
-        private static ProfilerMarker s_TransportDisconnect = new ProfilerMarker($"{nameof(NetworkManager)}.TransportDisconnect");
-#endif
-
-        private const double k_TimeSyncFrequency = 1.0d; // sync every second
-        private const float k_DefaultBufferSizeSec = 0.05f; // todo talk with UX/Product, find good default value for this
-
-        internal static string PrefabDebugHelper(NetworkPrefab networkPrefab)
+        // TODO 2023-Q2: Team discussion
```

Comment:
these types of comments are promising a timeline inline in the codebase which we do not want to be liable for necessarily.

Reply:
They are to be removed after the final review before merging.  Just labeled in a way that was easy to find.